godtools/App/Features/Menu/Presentation/Account/Subviews/AccountActivityView/Subviews/AccountActivityStatView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AccountActivityStatView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 3/20/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct AccountActivityStatView: View {
+    
+    let stat: UserActivityStatDomainModel
+    
+    var body: some View {
+        
+        HStack {
+            
+            Image(stat.iconImageName)
```

Comment:
What are your thoughts on ensuring the Image size stays consistent with the Figma design by adding additional modifiers?

By default SwiftUI Image will size itself based on the asset which would cause problems if we add additional assets that are larger.

https://swiftwithmajid.com/2020/05/27/mastering-images-in-swiftui/

Reply:
Yeah that's fair.  Ideally we'd get images that are sized correctly to begin with, but from past experience that's sometimes harder than it sounds 😄